File: Transport/OtherModels/tdbf/Fuel.cs
using Kdn.Ext.Attributes;

namespace Transport.OtherModels.tdbf
{
    public class Fuel
    {
        [IdProperty]
        public int fuelId { get; set; }
        public string fuelName { get; set; }
        public decimal k_weight { get; set; }
    }



}

File: Transport/JS/Transport/combo/Fuel.js
T.combo.Fuel = Ext.extend(Kdn.form.ComboBox, {
    initComponent: function() {
        Ext.applyIf(this, {
            valueField: 'FuelId',
            displayField: 'FuelName',            
            store: Kdn.ModelFactory.getStore('Fuel')
        });
        T.combo.Fuel.superclass.initComponent.call(this);
    }
});

Ext.reg('combo.fuel', T.combo.Fuel);